engine/class_modules/sc_death_knight.cpp

```diff
@@ -4364,6 +4361,12 @@ struct death_coil_t : public death_knight_spell_t
     p() -> cooldown.dark_transformation -> adjust( -timespan_t::from_seconds(
       p() -> spec.death_coil -> effectN( 2 ).base_value() ) );
 
+    // Reduces the cooldown on Apocalypse by 1.5s
```

Comment:
With the return of DT, Apocalypse doesn't look like it's benefitting from death coil rank 2 on PTR.

Reply:
Yeah I'm gonna check beta as well but it looks like death coil 2 now applies to DT instead of Apoc.  I'm gonna go ahead and finish out the DT and Apoc changes/ranks today since it looks like they're maybe done with the reworks.